src/XMakeBuildEngine/Resources/Strings.resx

```diff
@@ -436,6 +436,10 @@
     <comment>{StrBegin="MSB4020: "}UE: This is a generic message that is displayed when we find a project element with an incorrect value for one of its
     attributes e.g. &lt;Import Project=""&gt; -- the value of Project should not be an empty string.</comment>
   </data>
+  <data name="InvalidAttributeExclusive" UESanitized="false" Visibility="Public">
+    <!-- TODO: Replace ???? with a real error code -->
+    <value>MSB????: At most one of the include, remove, and update attributes may be specified for an item element.</value>
```

Comment:
Can you make sure this gets done please? We don't want to end up localizing this or leaving it in.